apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/HandlingerForOppgave.tsx

```diff
@@ -47,6 +47,7 @@ export const HandlingerForOppgave = ({ oppgave }: { oppgave: OppgaveDTO }) => {
           </Button>
         </>
       )
+    case 'MANUELL_JOURNALFOERING':
```

Comment:
Denne burde ikke ligge her? Da får man en lenke som ikke fungerer for manuell journalføring

Reply:
Sant det. Fikset ✅